src/components/Modal/staking-hub/AddAddressToERC1820RegistryModal.tsx

```
@@ -0,0 +1,217 @@
+
+  const handleClick = () => {
+    set_txStarted(true);
+    writeContract?.(data!.request);
+  };
+
+  const handleCloseModal = () => {
+    closeModal();
+  };
+
+  const startRefetchResult = async () => {
+    while(true) {
```

ESLint

[error] Unexpected constant condition.

(no-constant-condition)